modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -166,7 +160,7 @@ def self.consume_body_tempfile(lines, separator)
         end
         linechomp = line.chomp(LINE_BREAK)
         case linechomp
-        when "#{separator}--", "#{separator}--#{CARRIAGE_RETURN}"
+        when "#{separator}--"
```

Comment:
These carriage return cases were not needed as they were only added for `StringIO` support, which is being removed. (See #5941 for more info on when they were added.)